### PR TITLE
protect multi subscribe on chart

### PR DESCRIPTION
**Pull request by klee0kai**

### CODE REVIEW

File: Sources/LightweightCharts/Implementations/API/Chart.swift [hunk 1]
```diff
@@ -54,17 +60,28 @@ class Chart: JavaScriptObject {
     }
     
     private func subscribe(subscription: Subscription) {
+        if (activeSubscriptions[subscription] == .active){
+            return
```
Comment (makedonsky94) [inline Sources/LightweightCharts/Implementations/API/Chart.swift:64]: log error pls

File: Sources/LightweightCharts/Implementations/API/Chart.swift [hunk 2]
```diff
@@ -54,17 +60,28 @@ class Chart: JavaScriptObject {
     }
     
     private func subscribe(subscription: Subscription) {
+        if (activeSubscriptions[subscription] == .active){
```
Comment (makedonsky94) [inline Sources/LightweightCharts/Implementations/API/Chart.swift:63]:
```suggestion
        if (activeSubscriptions[subscription] == .active) {
```

File: Sources/LightweightCharts/Implementations/API/Chart.swift [hunk 3]
```diff
@@ -54,17 +60,28 @@ class Chart: JavaScriptObject {
...
+            context.addMessageHandler(messageHandler, name: name)
+        }
         let script = subscriberScript + "\n\(jsName).subscribe\(subscription.jsRepresentation)(\(name));"
-        context.addMessageHandler(messageHandler, name: name)
         context.evaluateScript(script, completion: nil)
+        activeSubscriptions[subscription] = .active
     }
     
     private func unsubscribe(subsription: Subscription) {
+        if (activeSubscriptions[subsription] != .active ){
```
Comment (makedonsky94) [inline Sources/LightweightCharts/Implementations/API/Chart.swift:78]:
```suggestion
        if (activeSubscriptions[subsription] != .active) {
```

File: Sources/LightweightCharts/Implementations/API/Chart.swift [hunk 4]
```diff
@@ -54,17 +60,28 @@ class Chart: JavaScriptObject {
...
+            return
```
Comment (makedonsky94) [inline Sources/LightweightCharts/Implementations/API/Chart.swift:79]: We need error log here too

**Review by makedonsky94 — CHANGES_REQUESTED** [review]

File: Sources/LightweightCharts/Implementations/API/Chart.swift (hunk 1 above)
Comment (klee0kai) [reply Sources/LightweightCharts/Implementations/API/Chart.swift:65]: done

File: Sources/LightweightCharts/Implementations/API/Chart.swift (hunk 2 above)
Comment (klee0kai) [reply Sources/LightweightCharts/Implementations/API/Chart.swift]: done

File: Sources/LightweightCharts/Implementations/API/Chart.swift (hunk 3 above)
Comment (klee0kai) [reply Sources/LightweightCharts/Implementations/API/Chart.swift]: done

**Pull request merged**